TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -105,6 +105,16 @@ def makeTreeFromMiniAOD(
             isLHE = cms.bool(False)
         )
         VarsDouble.extend(['SusyScan:SusyMotherMass','SusyScan:SusyLSPMass'])
+
+        # pMSSM ID for identifying the pMSSM model point
+        from TreeMaker.Utils.pmssm_cfi import PmssmProducer
+        process.Pmssm = PmssmProducer.clone(
+            shouldScan = cms.bool(fastsim),
```

Comment:
I don't think we want to run this for all fastsim. Maybe we should add a separate pmssm parameter to makeTree and runMakeTree.